lib/Net/DRI.pm

```diff
@@ -284,6 +284,7 @@ sub new
      'Verisign::NAME' => 'VeriSign::NameStore',
      'VNDS' => 'VeriSign::COM_NET', # warning, this was actually split into COM_NET and NameStore!
      'WS' => 'GDI',
+     'Afilias::Afilias' => 'Donuts' # TLds moved to Donuts but extentions only work if using Afilias::Shared
```

Comment:
What does this mean actually? Are the Afilias extensions supported in Donuts?

This is also a messy place to put these comments, if any explanation is required it should really be in the DRD file, but I probably wouldn't overcomplicate it and just ignore unless its critical :)

Reply:
No, Afilias extensions are not suported in Donuts.Ill remove the comment and put it in the DRD.